src/Command/InfectionCommand.php

```diff
@@ -481,38 +487,68 @@ private function getTestFrameworkExtraOptions(string $testFrameworkKey): TestFra
     protected function initialize(InputInterface $input, OutputInterface $output)
     {
         parent::initialize($input, $output);
+
         $locator = $this->getContainer()->get('locator');
+
         if ($customConfigPath = $input->getOption('configuration')) {
             $locator->locate($customConfigPath);
-            $this->doInitialize($input);
+        } else {
+            $this->runConfigurationCommand($locator);
+        }
 
-            return;
+        $this->io = $this->getApplication()->getIO();
+        $this->skipCoverage = \strlen(trim($input->getOption('coverage'))) > 0;
+        $this->eventDispatcher = $this->getContainer()->get('dispatcher');
+    }
+
+    public function isDebuggerOrAdditionalParametersExist(): bool
+    {
+        if (!$this->skipCoverage
+            && PHP_SAPI !== 'phpdbg'
+            && !\extension_loaded('xdebug')
+            && !XdebugHandler::getSkippedVersion()
+            && !$this->isInitialTestPhpOptionHasXdebug()
+        ) {
+            $this->io->error([
+                'Neither phpdbg or xdebug has been found. One of those is required by Infection in order to generate coverage data. Either:',
+                '- Enable xdebug and run infection again' . PHP_EOL .
+                '- Use phpdbg: phpdbg -qrr infection' . PHP_EOL .
+                '- Use --coverage option with path to the existing coverage report' . PHP_EOL .
+                '- Use --initial-tests-php-options option with `-d zend_extension=xdebug.so` and/or any extra php parameters',
+            ]);
+
+            return false;
         }
 
+        return true;
+    }
+
+    private function runConfigurationCommand(Locator $locator)
+    {
         try {
             $locator->locateAnyOf(InfectionConfig::POSSIBLE_CONFIG_FILE_NAMES);
         } catch (\Exception $e) {
             $configureCommand = $this->getApplication()->find('configure');
 
             $args = [
-                '--test-framework' => $input->getOption('test-framework'),
+                '--test-framework' => $this->input->getOption('test-framework'),
             ];
 
             $newInput = new ArrayInput($args);
-            $newInput->setInteractive($input->isInteractive());
-            $result = $configureCommand->run($newInput, $output);
+            $newInput->setInteractive($this->input->isInteractive());
+            $result = $configureCommand->run($newInput, $this->output);
 
             if ($result !== 0) {
                 throw InfectionException::configurationAborted();
             }
         }
-        $this->doInitialize($input);
     }
 
-    private function doInitialize(InputInterface $input)
+    private function isInitialTestPhpOptionHasXdebug(): bool
```

Comment:
it should be either *is* or *has* 

e.g. `initialTestPhpOptionsHaveXdebug()` or `isXdebugIncludedInInitialTestPhpOptions()` 

(I'd side with the first for English-readability, but the latter is fine too)